tests/php/tests/includes/helper/test_class.wp-job-manager-com-auth-token.php

```diff
@@ -0,0 +1,263 @@
+<?php
+
+require_once JOB_MANAGER_PLUGIN_DIR . '/includes/helper/class-wp-job-manager-com-auth-token.php';
+
+/**
+ * @group helper
+ * @group helper-base
+ */
+class WP_Test_WP_Job_Manager_Com_Auth_Token extends WPJM_BaseTest {
+
+	public function testInstance_WhenCalled_ReturnSameInstance() {
```

Comment:
Not sure if I misunderstood something, but aren't these 2 tests with the name exchanged? (`testInstance_WhenCalled_ReturnSameInstance` and `testInstance_WhenCalled_ReturnCorrectType`)

Reply:
One tests if the instance returned is the same, another tests if it returns the correct type.

IMHO it is kinda a boilerplate test, as it is testing..the singleton (which isn't technically even needed by this class), but oh well